tests/src/test/java/alluxio/server/tieredstore/LostStorageIntegrationTest.java

```diff
@@ -60,9 +78,21 @@ public void reportLostStorageInWorkerRegister() throws Exception {
     File hddDir = Files.createTempDir();
     String hddPath = hddDir.getAbsolutePath();
 
-    // Set to read only so worker storage paths cannot be initialize
-    ssdDir.setReadOnly();
-    hddDir.setReadOnly();
+//    // Mock no write permission so worker storage paths cannot be initialize
+    PowerMockito.mockStatic(StorageDir.class);
+    Mockito.when(StorageDir.newStorageDir(any(StorageTier.class),
+        anyInt(),
+        anyLong(),
+        anyLong(),
+        anyString(),
+        anyString())).thenCallRealMethod();
+    Mockito.when(StorageDir.newStorageDir(any(StorageTier.class),
+        anyInt(),
+        anyLong(),
+        anyLong(),
+        or(startsWith(ssdPath), startsWith(hddPath)),
```

Comment:
Why need two mockito mock? 
can the path be strings other than ssdPath and hdd path?

Reply:
Yeah, the mem tier path is another path, we need to call real method when the path is mem path.